### PR TITLE
Pin Skia version to 87.* and explain why

### PR DESCRIPTION
**Pull request by pavpanchekha**
I've discussed this with the Skia-Python maintainers and I think the conclusion we've come to (I'll send them this PR and ask if they agree!) is to release a new 87.6 release that supports Python 3.12 but otherwise has minimal changes from 87.5, and to pin that version in the book. To that end, I've added a paragraph about the version issue, asked users to check recent Skia versions before filing bugs, and added a new page about porting (not linked from the table of contents so that it's not included in the printed book, but which we can link after that's done).

@HinTak, @kyamagu, hopefully you agree with the text, if not, there's about a week to change it.

### CODE REVIEW

File: book/porting.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+87, Tk 8.6.14, DukPy 0.3.0, and PySDL2 0.9.15. This page documents
+code changes necessary to port the code of *Web Browser Engineering*
+to the most recent release of each library. It will be regularly
+updated as new versions are released and tested.
+
+Porting to Skia 124
+-------------------
+
+Skia 124, and the associated skia-python 124b6 release, changes a
+could of APIs used in this book. One major change is the removal of
```
Comment (chrishtr) [inline book/porting.md:16]:
```suggestion
couple of APIs used in this book. One major change is the removal of
```

File: book/porting.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+`FilterQuality`, used in [Chapter 15](embeds.md), and its replacement
+by `SamplingOptions`. This requires replacing `parse_image_rendering`
```
Comment (chrishtr) [inline book/porting.md:18]:
```suggestion
by `SamplingOptions`. This requires updating `parse_image_rendering`
```

File: book/porting.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+``` {.python}
+class DrawImage:
+    def execute(self, canvas):
+        canvas.drawImageRect(self.image, self.rect, self.quality)
+```
+
+The new API is somewhat cleaner and more customizable.
+ 
+Additionally, this release has difficulties drawing anti-aliased text
```
Comment (chrishtr) [inline book/porting.md:41]:
```suggestion
Additionally, as of our latest testing this release has difficulties drawing anti-aliased text
```

**Review by chrishtr — APPROVED** [review]

**Comment by HinTak** [comment]
v87.6 is out (should land in pypi in a few hours), mainly it is a build-matrix update for build hosts, python versions, github ci actions. functionally, differs from v87.5 by only two typos.

**Pull request merged**